src/main/java/com/kamesuta/bungeepteropower/Config.java

```diff
@@ -29,6 +29,11 @@ public class Config {
      * the server will be stopped after this time has elapsed according to the timeout setting.
      */
     public final int startTimeout;
+    /**
+     * The number of seconds the plugin will try to connect the player to the desired server
+     * Set this to the maximum time the server can take to start
+     */
+    public final int autoJoinTimeout;

```

Comment:
The name autojoin is confusing with the name of the function that starts when you try to enter the server and should be changed.  
How about "startupJoin"?